contribution_process.md

```diff
@@ -0,0 +1,164 @@
+opened prior to any pull request that implements a change to resolve the
+request. The GitHub issue captures information necessary to make the
+contribution, such as:
+
+1. A description of the requested change.
+1. The number of users in the community that request the change, measured
+   through GitHub :+1: reactions.
+1. Design discussions, including use cases, edge cases, and alternative
+   solutions.
+
+Maintainers regularly review open GitHub issues and label the ones that are
+ready for community contribution with the `community` label and add them to the
+"Ready to contribute" lane of the project board.
+
+The decision to label an issue as ready for community contribution is ultimately
+up to the discretion of the maintainers. Depending on the issue type (bug or
+feature request) and area (code or documentation), the maintainers use a
+set of criteria to decide if the issue is ready for contribution. This may
+include:
+
+1. The change is feasible for a community member to contribute. See below for
+   more details.
+1. Feature requests should have at least 10 :+1: reactions to demonstrate impact
+   beyond the requester.
+1. The maintainers agree with the implementation plan, including any design
+   discussions (e.g., public interface design, edge cases, backwards
+   compatibility concerns).
+
+If the criteria are not satisfied, the maintainers should explicitly comment
+what criteria are not satsified and how the community can satisfy them. All
+criteria must be met prior to making a contribution so that feedback
+during the review process focuses on the implementation of the agreed upon
+design.
+
+Issues that do not receive enough :+1: reactions and do not have any activty for
+a year are considered stale. Without further interaction, they will be closed.
+
+Some changes are not open for community contribution. They may pose a security risk, be part of the code that is dynamically generated, may require interaction with internal teams to implement, or may require knowledge of the codebase that is not well documented. These include (but are not limited to):
+
+1. Changes to build processes. Some of our build processes are available through GitHub, but some are not. We need to carefully test any potential changes to be sure that they are compatible with all of our systems.
+1. GitHub Actions. Changes to our actions need to be made and reviewed with extra scrutiny for security purposes.
+1. Changes to models, including waiters, paginators, and resources. These requests are cross-SDK and managed by service teams so that all SDKs can use them with uniform behavior.
+1. Changes to configuration or credential files or processes. These changes can also affect the behavior in other SDKs and must thus be made in coordination with internal teams.
+
+## Contribution-ready stage
+
+A GitHub issue enters the contribution ready stage when a maintainer adds the
+issue to the "Ready to contribute" lane of the public [GitHub project
```

Comment:
Does it require the `community` label as well?

Reply:
Yes, we state it earlier in the intake stage section:

https://github.com/aws/aws-cli/pull/7015/files#diff-4b232fc56a55385ae97bf89c4aa293258182115a4d71f55787276218c1678c92R51

Reply:
added it here as well